Check fails with:

	ld: unrecognized option '-Wl,--as-needed'
	ld: use the --help option for usage information

diff -upr phpspy-0.5.0.orig/Makefile phpspy-0.5.0/Makefile
--- phpspy-0.5.0.orig/Makefile	2020-05-31 18:34:19.931393581 +0200
+++ phpspy-0.5.0/Makefile	2020-05-31 18:34:23.341389992 +0200
@@ -33,7 +33,6 @@ phpspy_static: $(wildcard *.c *.h) vendo
 	$(CC) $(phpspy_cflags) $(phpspy_includes) $(termbox_inlcudes) $(phpspy_defines) $(phpspy_sources) -o phpspy $(phpspy_ldflags) $(phpspy_libs) $(termbox_libs)
 
 phpspy_dynamic: $(wildcard *.c *.h)
-	@$(or $(has_termbox), $(error Need libtermbox. Hint: try `make phpspy_static`))
 	$(CC) $(phpspy_cflags) $(phpspy_includes) $(phpspy_defines) $(phpspy_sources) -o phpspy $(phpspy_ldflags) $(phpspy_libs) -ltermbox
 
 vendor/termbox/build/src/libtermbox.a: vendor/termbox/waf
